cmd/trust-manager/app/options/options.go

```diff
@@ -156,6 +156,10 @@ func (o *Options) addBundleFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.Bundle.SecretTargetsEnabled,
 		"secret-targets-enabled", false,
 		"Controls if secret targets are enabled in the Bundle API.")
+
+	fs.StringVar(&o.Bundle.PKCS12Password,
```

Comment:
@erikgb if this option is set at the controller-level, changing this value will cause all the newly reconciled bundles to have a password (not the existing ones). Only when an existing store is changed, the password will be set too.

Reply:
I would vote for an additional output-format (eg. PKCS12WithPassword)

Reply:
> if this option is set at the controller-level, changing this value will cause all the newly reconciled bundles to have a password (not the existing ones). Only when an existing store is changed, the password will be set too.

Why? We could make it happen. You get events for all watched resource types on startup.

Reply:
Could it be clarified in the flag description that this will change the passwords of all existing PKCS12 stores (and thus break existing setups)?
Also, let's fix any issues with mixed passwords in this PR.

Reply:
> let's fix any issues with mixed passwords in this PR.

Which mix?

Reply:
If you set the password to "password1" and change it to "password2" later, some stores will still have the "password1" and only newly reconciled stores will have "password2".

Reply:
> If you set the password to "password1" and change it to "password2" later, some stores will still have the "password1" and only newly reconciled stores will have "password2".

You can't update password without re-starting controller pod, once restarted, all stores will be reassembled with new password reflection.

Reply:
Did you test that all stores were updated?

Reply:
> Did you test that all stores were updated?

Yes, I did.
I've just checked once again by changing related _trust-manager_ Deployment resource property.